packages/selected-network-controller/CHANGELOG.md

```diff
@@ -5,8 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [4.1.0]
 ### Added
 - Add `SelectedNetworkMiddlewareJsonRpcRequest` type ([#1970](https://github.com/MetaMask/core/pull/1970)).
+- Add method to reset proxies when flag toggled ([#3593](https://github.com/MetaMask/core/pull/3593)).
```

Comment:
What do you think about naming the method here? This would save people from having to look in the PR.